dev/archery/archery/integration/datagen.py

```diff
@@ -1461,6 +1526,16 @@ def generate_recursive_nested_case():
     return _generate_file("recursive_nested", fields, batch_sizes)
 
 
+def generate_ree_case():
+    fields = [
+        RunEndEncodedField('ree16', 16, get_field('values', 'int32')),
+        RunEndEncodedField('ree32', 32, get_field('values', 'utf8')),
+        RunEndEncodedField('ree64', 64, get_field('values', 'float32')),
+    ]
+    batch_sizes = [0, 7, 10]
+    return _generate_file("ree", fields, batch_sizes)
```

Comment:
Let's avoid cryptic file names.
```suggestion
    return _generate_file("run_end_encoded", fields, batch_sizes)
```